frontend/app/components/input-or-show/template.hbs

```diff
@@ -1,14 +1,14 @@
 {{#if edit}}
   {{#if multiline}}
-    {{textarea class="form-control" value=value placeholder=placeholder}}
+    {{textarea class='uk-textarea' value=value placeholder=placeholder}}
   {{else}}
-    {{input class="form-control" value=value placeholder=placeholder}}
+    {{input class='uk-input' value=value placeholder=placeholder}}
   {{/if}}
 {{else}}
   {{#with (if mask "********" value) as |_value|}}
     {{#if clipboard}}
-      <a href="javascript:void(0)" {{action "copy"}} title="Copy to clipboard">
-        <i class="fa fa-clipboard"></i> <span class="value">{{trim _value}}</span>
+      <a href="#" {{action "copy"}} title="Copy to clipboard">
```

Comment:
Doesn't `href="#"` scroll all the way up?

Reply:
Doesn't matter - the `action` helper triggers an `event.preventDefault()`

Reply:
Ah ok :)

Reply:
And I think `javascript:void(0)` is kind of 1990...

Reply:
^^ right